### PR TITLE
Milestone #4

### PR DESCRIPTION
**Pull request by lucasmazo32**

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      newTotal = 0;
+      newNext = 0;
+      break;
+    case '%':
+      newTotal /= 100;
+      newNext /= 100;
+      break;
+    default:
+      newTotal = operate(newTotal, newNext, operation);
+      break;
```
Comment (azdrenymeri) [inline src/logic/calculate.js]: The case when `btnName == "."` is not covered :thinking:

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/extensions */
+
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const num1 = Big(numberOne);
+  const num2 = Big(numberTwo);
+  switch (operation) {
+    case '÷':
+      return num1 / num2;
```
Comment (azdrenymeri) [inline src/logic/operate.js]: If `num2` is zero(0) this will throw a `DivisionByZero` error.
Kindly cover this.

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
### Hello @lucasmazo32 :wave: 
#### Great job on this milestone :+1: 
#### Kindly implement the changes and add it for another review.

**Comment by lucasmazo32** [comment]
- Added the case for '.'
- Fixed / functionality.

**Pull request closed without merging**